galaxy.yml

```diff
@@ -1,7 +1,7 @@
 ---
 namespace: arillso
 name: test_do_not_use
```

Comment:
is this correct?

Reply:
Yes is correct, I have to test it if it works and this is the best name for it.